pkgs/development/libraries/libgnurl/default.nix

```diff
@@ -1,38 +1,30 @@
-{ stdenv, fetchurl, autoreconfHook, perl, zlib, gnutls, gss, openssl
-, libidn }:
+{ stdenv, fetchurl, libtool, groff, perl, pkgconfig, python2, zlib, gnutls, libidn }:
 
 stdenv.mkDerivation rec {
-  version = "7.54.1";
+  version = "7.61.1";
 
   name = "libgnurl-${version}";
 
   src = fetchurl {
-    url = "https://gnunet.org/sites/default/files/gnurl-${version}.tar.bz2";
-    sha256 = "0szbj352h95sgc9kbx9wzkgjksmg3g5k6cvlc7hz3wrbdh5gb0a4";
+    url = "https://ftp.gnu.org/gnu/gnunet/gnurl-${version}.tar.gz";
```

Comment:
Please use `url = "mirror://gnu/gnunet/..."` here. This automatically uses a mirror list provided in nixpkgs, see [here](https://github.com/NixOS/nixpkgs/blob/d35c7f9a914c17fcca4839593250d906de1afa24/pkgs/build-support/fetchurl/mirrors.nix#L29).